buddyboss/buddyboss-platform
# Elementor conflict parent forum is not getting saved - issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2340  .

### How to test the changes in this Pull Request:

1. Please install and activate elementor plugin & classic editor plugin
2. Then please see the below video

### Proof Screenshots or Video

https://www.loom.com/share/76e45692969b4053a388415b74a4f55e

### Other information:




### Changelog entry

> Elementor conflict parent forum is not getting saved - issue fixed
